framework/db/ActiveRecord.php

```diff
@@ -268,7 +268,7 @@ public static function deleteAll($condition = '', $params = [])
      */
     public static function find()
     {
-        return Yii::createObject(ActiveQuery::className(), [get_called_class()]);
+        return Yii::createObject(ActiveQuery::class, [get_called_class()]);
```

Comment:
`get_called_class()` is necessary as well since it could be inherited from `ActiveQuery`.